events/index.html

```diff
@@ -6,8 +6,8 @@
 notice: true
 ---
 
-{% assign now =  site.time |  date_to_xmlschema %}
-{% assign future_events = site.events | where_exp: "event","event.eventdate  > now" %}
+{% assign now = site.time %}
+{% assign future_events = site.events | where_exp: "event","event.eventdate > now" %}
```

Comment:
By using a parsable date for `event.eventdate`, we can use native comparison rather than text comparison to filter / sort dates.